rustls/examples/internal/test_ca.rs

```diff
@@ -8,6 +8,7 @@ use std::str::FromStr;
 use std::sync::atomic::{AtomicU64, Ordering};
 use std::time::Duration;
 
+use pki_types::SubjectPublicKeyInfoDer;
```

Comment:
I expect this commit is pretty uncontroversial -- let's split it off as a separate PR and merge that first. That should reduce the noise in this one.

Reply:
Great idea! Removed this commit from this PR and created a https://github.com/rustls/rustls/pull/2078. When that PR get's merged I'll do a rebase.